scripts/install.sh

```diff
@@ -328,7 +328,7 @@ function waitRunningContainer(){
 }
 
 function checkComposeVersion(){
-        compose_vers=$(docker-compose -v | grep -oP '\d+.\d+.\d+')
+        compose_vers=$(docker-compose -v | awk '{print $3}' | sed 's/\,//g')
```

Comment:
This is what I get with this command

─$ docker-compose -v | awk '{print $3}' | sed 's/\,//g'
version